lib/internal/async_hooks.js

```diff
@@ -82,7 +82,7 @@ const active_hooks = {
 
 const { registerDestroyHook } = async_wrap;
 const { enqueueMicrotask } = internalBinding('task_queue');
-const { resource_symbol, owner_symbol } = internalBinding('symbols');
```

Comment:
Does this mean that this symbol can be removed from `env.h` now?

Reply:
Yes, I have removed it from `env.h`.